lib/capistrano/tasks/nvm.rake

```diff
@@ -9,17 +9,21 @@ namespace :nvm do
 
       nvm_node_path = fetch(:nvm_node_path)
       nvm_node_path = [nvm_node_path] unless nvm_node_path.is_a?(Array)
+      nvm_verb = fetch(:nvm_verb).to_s
```

Comment:
Can you add validation for the verb, since it can only be `:use` or `:install` right? But then if there are only 2 options, maybe it would be better to make it a boolean property, e.g. `:nvm_install`, which when `true` causes the "verb" to be `install` instead of `use`

Reply:
>Can you add validation for the verb,

Absolutely.

>since it can only be `:use` or `:install` right?

Effectively yes, but let's discuss your other point...

>But then again, if there are only 2 options, maybe it would be better to make it a boolean property, e.g. :nvm_install, which when true causes the "verb" to be install instead of use

Hmm... I'd need to rewrite some things to kind of unify the behavior of this code. For example, when specifying `use` instead of `install`, the node version availability check is path based, so the version string has to be exact and must prepend the `v`, but it really should be running `nvm version` on the destination machines instead. 

I also _personally_ think that `:nvm_install` behavior should default to `true` as nvm is just a lot more useful that way, at least from an automation standpoint. That's technically a major version change so I guess if I really think that's the right call I can always submit another PR.

...I've got a good plan for how to handle this, but I suppose I needed to "think out loud" to get the ideas right. LMK if anything doesn't seem right to you, otherwise I'll work on this sometime in the coming days and get that done.